imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -42,9 +57,14 @@
 @Service("cdrFileService")
 public class CdrFileServiceImpl implements CdrFileService {
 
+    private static final String CDR_FILE_NOTIFICATION_URL = "imi.cdr_file_notification_url";
     private static final String CDR_FILE_DIRECTORY = "imi.cdr_file_directory";
+    private static final String MAX_CDR_ERROR_COUNT = "imi.max_cdr_error_count";
+    private static final int MAX_CDR_ERROR_COUNT_DEFAULT = 100;
     private static final String PROCESS_SUMMARY_RECORD = "nms.imi.kk.process_summary_record";
     private static final String CSR_PARAM_KEY = "csr";
+    private static final String PROCESS_DETAIL_FILE = "nms.imi.kk.process_detail_file";
```

Comment:
Add _SUBJECT or something to make it clear that this is a subject, not a config key.